packages/react-devtools-shared/src/hooks/SourceMapConsumer.js

```diff
@@ -229,6 +240,11 @@ function IndexedSourceMapConsumer(sourceMapJSON: IndexSourceMap) {
       );
     }
 
+    if (section.sourceMapConsumer === null) {
```

Comment:
nit: use != instead of === ?

Reply:
Sorry, what do you mean?

Reply:
Do you mean `==` instead of `===`?

In this case, it's always ether `null` or a `SourceMapConsumer` instance. (Flow typings ensure this too.)